srcpkgs/e2guardian/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'e2guardian'
+pkgname=e2guardian
+version=5.3.2
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --enable-clamd=yes --with-proxyuser=_e2guardian --with-proxygroup=_e2guardian --sysconfdir=/etc --localstatedir=/var --enable-icap=yes --enable-commandline=yes --enable-email=yes --enable-ntlm=yes --mandir=${prefix}/share/man --infodir=${prefix}/share/info --enable-pcre=yes --enable-sslmitm=yes"
+conf_files="/etc/e2guardian/*.conf
+/etc/e2guardian/site.story
+/etc/e2guardian/preauth.story
+/etc/e2guardian/site.story
+/etc/e2guardian/lists/authplugins/*
+/etc/e2guardian/lists/bannedrooms/*
+/etc/e2guardian/lists/contentscanners/*
+/etc/e2guardian/lists/phraselists/*/*
+/etc/e2guardian/authplugins/*
+/etc/e2guardian/contentscanners/*
+/etc/e2guardian/downloadmanagers/default.conf"
+make_dirs="/var/log/e2guardian 0755 _e2guardian _e2guardian"
+hostmakedepends="automake pkg-config"
+makedepends="zlib-devel pcre-devel libressl-devel"
+short_desc="E2guardian is a web content filter proxy with multiple server modes"
+maintainer="Philipp Nassua <philipp@nassua.cc>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/e2guardian/e2guardian"
+distfiles="https://github.com/e2guardian/e2guardian/archive/v${version}.tar.gz"
+checksum=ff15090157b8ea7b7910a893bb3bf582b5658cd2b4a9417805bd5c8cc1ba5e29
+system_accounts="_e2guardian"
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*)
+		configure_args+="-mno-sse2"
+		;;
+	i686*)
+		configure_args+="-mno-sse2"
```

Comment:
why is this necessary?

Reply:
I took the configure_args from the installation instructions of the project. I'm not sure why it would be necessary, so I could remove it.

Reply:
I just checked again, they have this flag only in the debian section of the INSTALL instructions, not for any other distribution. I just removed it